views/dashboard/partials/modal-dashboard-help.html.twig

```diff
@@ -83,10 +83,10 @@
                 <strong>Registered:</strong> A person who created an account by entering a mobile phone number or an email address but has not yet completed the informed consent process. A registered individual has a unique AoURP participant identifier (PID).
               </li>
               <li>
-                <strong>Participant:</strong> A person who has completed primary consent. Once the Registered Individual becomes a Participant, they will have the opportunity to respond to the EHR or DV_EHR Sharing Consent. If the Participant agrees, they will move on to the next status. If the Participant says, “No” to EHR or DV_EHR Sharing, the status will remain “Participant” and the individual will be able to complete PPI modules but will not continue to transition to the Fully Consented or Core Participant status.
+                <strong>Participant:</strong> A person who has completed primary consent. Once the Registered Individual becomes a Participant, they will have the opportunity to respond to the EHR or DV_EHR Sharing Consent. If the Participant agrees, they will move on to the next status. If the Participant says, “No” to EHR or DV_EHR Sharing, the status will remain “Participant” and the individual will be able to complete PPI modules but will not continue to transition to the Participant + EHR Consent or Core Participant status.
               </li>
               <li>
-                <strong>Fully Consented (Participant + EHR):</strong> A participant with EHR consent has completed the primary informed consent process and the HIPAA Authorization/EHR consent (if available and where relevant, based on AoURP capacity to effectively search for and access records). This definition takes into account DV EHR sharing. A Participant with EHR Consent may complete research program activities such as completion of questionnaires or providing physical measurements and biospecimens, but will remain in the Participant+EHR Consent status until all initial steps of the program are completed, qualifying the participant for Core Participant status.
+                <strong>Participant + EHR Consent (Participant + EHR):</strong> A participant with EHR consent has completed the primary informed consent process and the HIPAA Authorization/EHR consent (if available and where relevant, based on AoURP capacity to effectively search for and access records). This definition takes into account DV EHR sharing. A Participant with EHR Consent may complete research program activities such as completion of questionnaires or providing physical measurements and biospecimens, but will remain in the Participant+EHR Consent status until all initial steps of the program are completed, qualifying the participant for Core Participant status.
```

Comment:
Do we still require text in parentheses?

Reply:
I wouldn't think so. I'll remove them. :+1: